docs/notes/deploy-kubernetes.rst

```diff
@@ -31,9 +43,11 @@ with ``helm``:
     helm repo add vineyard https://vineyard.oss-ap-southeast-1.aliyuncs.com/charts/
     helm install vineyard vineyard/vineyard
```

Comment:
I think it should be `helm install vineyard-operator vineyard/vineyard-operator`, as you have removed the existing helm charts.

Reply:
And you need to double check the content in `README.rst` as well.